cmd/restic/cmd_cache.go

```diff
@@ -118,10 +126,20 @@ func runCache(opts CacheOptions, gopts GlobalOptions, args []string) error {
 			old = "yes"
 		}
 
+		var size string
+		if !opts.NoSize {
+			bytes, err := dirSize(fmt.Sprintf("%s/%s", cachedir, entry.Name()))
```

Comment:
Use `filepath.Join(cachedir, entry.Name())`, that's the correct function and also works on Windows.